simulations/ContestSimulation.java

```diff
@@ -0,0 +1,192 @@
+package domjudge;
+
+import io.gatling.javaapi.core.*;
+import io.gatling.javaapi.http.*;
+import java.time.Duration;
+import java.io.*;
+import java.util.*;
+import java.util.stream.Stream;
+import java.util.function.*;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static io.gatling.javaapi.core.CoreDsl.*;
+import static io.gatling.javaapi.http.HttpDsl.*;
+
+
+public class ContestSimulation extends Simulation {
+    // A fixed password (see seeding/accounts.json)
+    String PASSWORD="secret";
+
+    // How many "actions" a team will take (actions are things like submitting, requesting clarification, viewing a scoreboard)
+    int NUM_TEAM_ACTIONS = 5;
+
+    // Maximum amount of time the scenario can run
+    int MAX_MINUTES = 5;
```

Comment:
Sets a cap for how long this will run (which also sets how long a spectator will keep refreshing for.